e2e/helpers/srcFetch.ts

```diff
@@ -0,0 +1,161 @@
+export const NETWORK_UNAVAILABLE_MESSAGE =
```

Comment:
The SynapseClient doesn't expose the `fetchWithExponentialTimeout` method or `doGet` etc. Replicated SRC code here, so new API calls (e.g. creating a user, deleting a message) can be made.

Reply:
What do you think about exposing SRC.SynapseClient fetchWithExponentialTimeout (and any other functions that we need)?  I'd hate to have fixes made to these SynapseClient functions (or srcFetch functions) and have them drift apart

Reply:
Agreed that there's potential for drift! Exposing SRC's HttpClient would remove the need for duplicated functions, since we would have access to `doGet`, `doPost`, etc... which use `fetchWithExponentialTimeout`. Would you like me to include those changes as part of this PR or split the work into a separate ticket? 

We would still need to duplicate a few types (LoginResponse, FileHandleStatus, FileHandle, PaginatedResults) until the synpase-types package is also exposed, as discussed with @nickgros [here](https://github.com/Sage-Bionetworks/SynapseWebClient/pull/5162/files#r1303339880) and associated ticket [here](https://sagebionetworks.jira.com/browse/SWC-6529).

Reply:
I created a [ticket](https://sagebionetworks.jira.com/browse/SWC-6531) to track exposing the HttpClient and removing the duplicate code.